spec/yaml/swiftnav/sbp/acquisition.yaml

```diff
@@ -70,3 +70,79 @@ definitions:
             desc: |
               PRN-1 identifier of the satellite signal for which
               acquisition was attempted
+
+ - AcqSvProfile:
+    short_desc: Reacq perfomance measurement and debug
+    desc: |
+      Profile for a specific SV for debugging purposes
+      The message describes SV profile during acquisition time.
+      The message is useful to debug and measure the performance of
+      the new (re-)acq design.
```

Comment:
Let's rephrase it a bit: 'The message is used to debug and measure the performance of acquisition.'